### DISCUSSION TITLE
code block preview instead of full code block

### DISCUSSION THREAD

**Discussion by stefan-berkner-tilotech** [retypeapp/retype#407] [Ideas]
Some parts of our documentation contain very long code examples - stuff that a normal reader wouldn't be interested in, but that should still be there for the nerds.

Is it somehow possible to only show a preview of the first few lines instead of the code block and to only display the full code block once the reader clicks on it? If not, would you consider such a thing as usefull and integrate it into retype?

**Comment by DannyBen** [comment]
Its a great idea. I would also find uses for it.

I am currently hiding long code under a collapsible panel - as can be seen [here](https://bashly.dannyb.co/configuration/command/), at the top of the page under the **Example** panel - but I feel it is quite easy to miss.

Would much rather show the first 5 lines of the example with a "Show more" button/link at the bottom of it.
